docker-wrappers/LocalNeighborhood/Dockerfile

```diff
@@ -1 +1,9 @@
-# Create a Docker image for the Local Neighborhood algorithm here
+#Builds base python container from python image on docker hub
+FROM python:3-alpine
```

Comment:
I typically use a base image that locks in a specific version of Python 3.x (e.g. 3.11 or 3.12) so the behavior is more stable over time.